src/tests/redirects.py

```diff
@@ -0,0 +1,35 @@
+# © 2024 Laserfiche.
+# See LICENSE-DOCUMENTATION and LICENSE-CODE in the project root for license information.
+import requests
+SCHEME_AND_DOMAIN = "https://developer.laserfiche.com"
+
+def getLinks(path):
+    links = []
+    with open(path, "r") as f:
+        lines = f.readlines()
+        links = [SCHEME_AND_DOMAIN + url[:-1] for url in lines]
+    return links
+
+def findBrokenLinks(links):
+    broken_links = {}
+
+    for url in links:
+        r = requests.get(url)
+        if (r.status_code != 200):
+            broken_links[url] = r.status_code
+    
+    return broken_links
+
+def fmtBrokenLinks(broken_links):
+    map_list = [ url + " -> " + str(broken[url]) for url in broken_links.keys()]
+    return "\n".join(map_list)
+
+fname = "./src/tests/redirected-links.txt"
+links = getLinks(fname)
+broken = findBrokenLinks(links)
+
+if len(broken):
+    link_info = fmtBrokenLinks(broken)
+    raise Exception(f"{len(broken)} link(s) returned a non-200 response:\n" + link_info)

```

Comment:
Any reasons for throwing an exception instead of print error message and exit 1?

Reply:
no reason, is this preferred?